job-server/config/shiro.ini

```diff
@@ -0,0 +1,16 @@
+
+# use this for basic ldap authorization, without group checking
+activeDirectoryRealm = org.apache.shiro.realm.ldap.JndiLdapRealm

```

Comment:
@velvia  sure, makes sense. How about this for the README.md then: 
1.1 Copy `config/local.sh.template` to `<environment>.sh` .....
1.2 Copy `config/shiro.ini.template` to `shiro.ini` and edit as appropriate. NOTE: only required when `authentication = on`